docs/sources/k6-studio/components/test-recorder.md

```diff
@@ -21,7 +21,7 @@ The Test Recorder window is composed of:
 3. **Test Recorder options**: Below the test recording name, you can see:
    - **Requests**: The total number of requests in the recording
    - **Show static assets**: A toggle that controls whether you can see all static assets requests in the Requests list. The static assets requests are hidden by default.
-   - **Filter**: A search box that lets you filter the list of requests by URL, method (such as GET or POST), and status code.
+   - **Search**: A search box that lets you filter the list of requests by URL, method (such as GET or POST), and status code. You can also use the toggle to search across all request data, such as headers, cookies, payload, and response data.
```

Comment:
Might be better to swap the message sentences here since by default we search all data 🤔